src/models/StandaloneSimilarity.py

```diff
@@ -0,0 +1,21 @@
+import random
+
+from cleaning_functions import pandas_vector_to_list
```

Comment:
Do we really need a function for this? Surely just use `list(df.itertuples())`? Maybe I'm misunderstanding.